pages/api/request/help.ts

```diff
@@ -0,0 +1,88 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
Unit tests are needed for this file. Since we're looking to address tech debt as of late it might be worth it so as we don't add more